worlds/metroidprime/config.py

```diff
@@ -1,1811 +1,1639 @@
+                "Artifact of Newborn": starting_inventory(world, "Artifact of Newborn")
+            },
+            "requiredArtifactCount": options.required_artifacts.value
+        },
+        "levelData": {
+            "Frigate Orpheon": {
+                  "transports": {
+                      "Frigate Escape Cutscene": "Tallon Overworld:Landing Site"
+                  },
+                  "rooms": {
+                        "Exterior Docking Hangar": {},
+                        "Air Lock": {},
+                        "Deck Alpha Access Hall": {},
+                        "Deck Alpha Mech Shaft": {},
+                        "Emergency Evacuation Area": {},
+                        "Connection Elevator to Deck Alpha": {},
+                        "Deck Alpha Umbilical Hall": {},
+                        "Biotech Research Area 2": {},
+                        "Map Facility": {},
+                        "Main Ventilation Shaft Section F": {},
+                        "Connection Elevator to Deck Beta": {},
+                        "Main Ventilation Shaft Section E": {},
+                        "Deck Beta Conduit Hall": {},
+                        "Main Ventilation Shaft Section D": {},
+                        "Biotech Research Area 1": {},
+                        "Main Ventilation Shaft Section C": {},
+                        "Deck Beta Security Hall": {},
+                        "Connection Elevator to Deck Beta (2)": {},
+                        "Subventilation Shaft Section A": {},
+                        "Main Ventilation Shaft Section B": {},
+                        "Biohazard Containment": {},
+                        "Deck Gamma Monitor Hall": {},
+                        "Subventilation Shaft Section B": {},
+                        "Main Ventilation Shaft Section A": {},
+                        "Deck Beta Transit Hall": {},
+                        "Reactor Core": {},
+                        "Cargo Freight Lift to Deck Gamma": {},
+                        "Reactor Core Entrance": {}
+                    },
+              },
+            "Tallon Overworld": {
+                    "transports": {
+                            "Tallon Overworld North (Tallon Canyon)":  "Chozo Ruins West (Main Plaza)",
+                            "Tallon Overworld West (Root Cave)": "Magmoor Caverns East (Twin Fires)",
+                            "Tallon Overworld East (Frigate Crash Site)": "Chozo Ruins East (Reflecting Pool, Save Station)",
+                            "Tallon Overworld South (Great Tree Hall, Upper)": "Phazon Mines East (Main Quarry)",
+                            "Tallon Overworld South (Great Tree Hall, Lower)": "Chozo Ruins South (Reflecting Pool, Far End)",
+                            "Artifact Temple": temple_dest(options.final_bosses)
+                    },
+                    "rooms": {
+                        "Landing Site": {
+                            "pickups": [{

```

Comment:
I learned that `pickups` needs to be an array of objects, you'll see a lot of updates for the rest of the file with that change guiding it.